src/gui/mainwindow.cpp

```diff
@@ -187,6 +189,16 @@ void MainWindow::neovimMaximized(bool set)
 	}
 }
 
+void MainWindow::neovimFrameless(bool set)
```

Comment:
```suggestion
void MainWindow::neovimFrameless(bool isFrameless)
```

nitpick: The name `set` is a little ambiguous.

Reply:
done